lib/gym/generators/package_command_generator_xcode7.rb

```diff
@@ -43,10 +43,17 @@ def temporary_output_path
       def ipa_path
         unless Gym.cache[:ipa_path]
           path = Dir[File.join(temporary_output_path, "*.ipa")].last
-          ErrorHandler.handle_empty_archive unless path
-
-          Gym.cache[:ipa_path] = File.join(temporary_output_path, "#{Gym.config[:output_name]}.ipa")
-          FileUtils.mv(path, Gym.cache[:ipa_path]) if File.expand_path(path).downcase != File.expand_path(Gym.cache[:ipa_path]).downcase
+          if path
+            Gym.cache[:ipa_path] = File.join(temporary_output_path, "#{Gym.config[:output_name]}.ipa")
+            FileUtils.mv(path, Gym.cache[:ipa_path]) if File.expand_path(path).downcase != File.expand_path(Gym.cache[:ipa_path]).downcase
+          elsif Dir.exist?(apps_path)
+            files = Dir[File.join(apps_path, "*.ipa")]
+            path = files.min_by(&:length)
```

Comment:
When we export IPA files without app thinning, the Xcode generates one IPA file inside the output folder. But when the app thinning is used, the Xcode creates `Apps` folder inside the output folder. All the IPA files (thinned and one generic) are stored inside the `Apps` directory. So
If the IPA file exists in the output directory (line 46), the "old" logic is used.
If the IPA file doesn't exists in the output directory (line 49), `gym` tries to find all IPA files in the `Apps` folder. One of those files is always the "generic" IPA file, it doesn't contain any suffix, so its name is the shortest one (line 51).

I hope this will be enough. Or, maybe I should add some comments?